drivers/sensor/nxp/nxp_mcx_temp/Kconfig

```diff
@@ -0,0 +1,17 @@
+# NXP MCX temperature sensor configuration options
+
+# Copyright (c) 2025 Vestas Wind Systems A/S
+# SPDX-License-Identifier: Apache-2.0
+
+# NXP MCX temperature sensor configuration options
+
+# Copyright (c) 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
this needs to be fixed, i'm not sure what's going on here. if you copied this file then made edits, then you need to put both copyrights, but not duplicate the whole header. Also, the NXP one should be 2025